tools/linter/adapters/mypy_linter.py

```diff
@@ -122,9 +122,12 @@ def check_files(
     retries: int,
     code: str,
 ) -> List[LintMessage]:
+    # dmypy has a bug where it won't pick up changes if you pass it absolute
+    # file names, see https://github.com/python/mypy/issues/16768
+    filenames = [os.path.relpath(f) for f in filenames]
```

Comment:
absolute filenames would be `os.path.abspath` right? Maybe clarify the comment?

Reply:
Err, you must pass RELATIVE paths, not ABSOLUTE paths. The absolute path case is buggy.